README.md

```diff
@@ -34,10 +35,14 @@ optional arguments:
                         Generate projects files for IDE. Options are: vscode
   -r, --runFromRAM      Run the program from RAM rather than flash
   -uart, --uart         Console output to UART (default)
+<<<<<<< HEAD
```

Comment:
Looks like there's still a merge issue here you need to (manually) fix up.

Reply:
:man_facepalming: done now